ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

```diff
@@ -147,6 +147,7 @@ Delete Data Science Projects From CLI
             Continue For Loop
         ELSE
             Oc Delete    kind=Project   name=${project_k8s_name}
+            Sleep    30s    reason=Wait for the project to be really deleted
```

Comment:
could you make it waiting for the project to be deleted instead of sleeping? Because time for delation may vary depending on the deployed resources. 30s sleep may be too much for certain cases.

Reply:
To be fair we've been running this with no check whatsoever for quite some time, even a dumb sleep is better than that :smile: 
But yeah I can come up with something